databricks_blueprints/upload_file_to_dbfs.py

```diff
@@ -0,0 +1,136 @@
+import sys
+import argparse
+import base64
+import os
+import re
+import requests
+import shipyard_utils as shipyard
+try:
+    import helpers
+    import errors
+except BaseException:
+    from . import helpers
+    from . import errors
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--instance-id', dest='instance_id', required=True)
+    parser.add_argument('--source-file-name',
+                        dest='source_file_name',
+                        required=True)
+    parser.add_argument('--source-folder-name',
+                        dest='source_folder_name',
+                        required=False)
+    parser.add_argument('--dest-file-name',
+                        dest='dest_file_name',
+                        required=False)
+    parser.add_argument('--dest-folder-name',
+                        dest='dest_folder_name',
+                        required=False)
+    parser.add_argument('--source-file-name-match-type',
+                        dest='source_file_name_match_type',
+                        choices={'exact_match', 'regex_match'},
+                        required=True)
+    args = parser.parse_args()
+    return args
+
+
+def upload_file_to_dbfs(client, local_file_path, dest_file_path):
+    # create handle for stream upload chunking (blocks)
+    create_handle_endpoint = "/dbfs/create"
+    payload = {"path": dest_file_path, "overwrite": "true"}
+    try:
+        handle_response = client.post(create_handle_endpoint, data=payload)
+    except Exception as e:
+        if 'nodename nor servname provided' in str(e):
```

Comment:
Is there no better way to determine what error is coming back - header, response code, error body, etc.? If there is let's use it, otherwise you can keep.